### PR TITLE
Fix lints in db subpackage

### PR DESCRIPTION
**Pull request by freeekanayaka**
This makes the db subpackage golint-clean.

```bash
free@x1:~/go/src/github.com/lxc/lxd$ golint lxd/db/
free@x1:~/go/src/github.com/lxc/lxd$ 
```

### CODE REVIEW

**Comment by freeekanayaka** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
This should also modify test/suites/static_analysis.sh to add a call for lxd/db so this can't regress in the future.

File: test/suites/static_analysis.sh [hunk 1]
```diff
@@ -68,9 +68,7 @@ test_static_analysis() {
       golint -set_exit_status lxd-p2c
 
       golint -set_exit_status lxd/config
-      golint -set_exit_status lxd/db/node
-      golint -set_exit_status lxd/db/query
-      golint -set_exit_status lxd/db/schema
```
Comment (stgraber) [inline test/suites/static_analysis.sh]: You shouldn't remove the sub-packages, golint isn't recursive, it only evaluates one package.

Comment (stgraber) [reply test/suites/static_analysis.sh]: Well, it can be made recursive but then it'd be "lxd/db/..." (untested).

Comment (freeekanayaka) [reply test/suites/static_analysis.sh]: Fixed.

**Comment by stgraber** [comment]
@brauner feel free to merge this one if Jenkins looks green enough.

**Pull request merged**